.github/workflows/main.yml

```diff
@@ -0,0 +1,116 @@
+name: Continuous Integration
+
+on: [push, pull_request]
+
+env:
+  # needed by coveralls
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  CIBW_BUILD: "cp36-* cp37-* cp38-*"
```

Comment:
No py35? No py39? Guess they already dropped py35, didn't they?

Reply:
Yeah, we'd have to use an older CIBuildWheel version. I know we had a discussion about keeping py35. You think we should stick to it?

Reply:
I think they still **build** wheels for py35, they just don't run CIBuildWheel under py35 AFAICT from their doc.

Reply:
Added both, py35 and py39